app/controllers/api/v1/reviews_controller.rb

```diff
@@ -3,4 +3,15 @@ class Api::V1::ReviewsController < ApplicationController
   def index
     render json: Review.all, adapter: :json
   end
+
+  def create
+    @review = Review.create!(review_params)
+    render json: @review, adapter: :json
+  end
```

Comment:
This needs a fail case if the review does not save! 
Also, ideally all references to `adapter` should be done in your Rails config. Please see the serializer reading from Week 6!